packages/calcite-components/src/components/accordion/accordion.tsx

```diff
@@ -132,6 +153,24 @@ export class Accordion {
   //
   //--------------------------------------------------------------------------
 
-  private sortItems = (items: any[]): any[] =>
-    items.sort((a, b) => a.position - b.position).map((a) => a.item);
+  private updateAccordionItems = (addedAccordionItem?: HTMLCalciteAccordionItemElement): void => {
+    if (addedAccordionItem) {
```

Comment:
What happens if an accordion item is removed from the DOM? Does it get removed from `accordionItems` somehow or will that cause issues?

Reply:
Nevermind it looks like it will get removed on line 163, but my question below still stands

Reply:
Although it looks like `updateAccordionItems()` exits early if there were no items added. So do we need to handle items being removed separately? I haven't spent a lot of time working in accordion so it's an open question, could be fine as is.